controls/srg_gpos/SRG-OS-000312-GPOS-00123.yml

```diff
@@ -5,5 +5,6 @@ controls:
         title: {{{ full_name }}} must allow operating system admins to grant their privileges
             to other operating system admins.
         rules:
-            - use_pam_wheel_for_su
+            - sysctl_fs_protected_symlinks
+            - sysctl_fs_protected_hardlinks
```

Comment:
Given the control title, it seems to me that `use_pam_wheel_for_su` makes more sense  than the rules added.
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_for_su/rule.yml#L30

Reply:
Could you clarify the reasoning? To me the symlink rules don't make sense to this SRG.

Reply:
When the controls have value 0, they symlink behavior is unrestricted  (https://sysctl-explorer.net/fs/protected_hardlinks/ https://sysctl-explorer.net/fs/protected_symlinks/)

I see the symlink syscontrols as a restriction mechanism rather then a "granting" or delegation mechanism, so I fail to see the connection.

If you have a strongly we can merge this, but I think we should avoid adding rules to SRGs with the link  /rationale  is weak.

Reply:
According to the [latest RHEL8 STIG](https://stigs.mab879.com/stigs/5/controls/RHEL-08-010373) these rules satisfy SRG-OS-000312-GPOS-00122, SRG-OS-000312-GPOS-00123, SRG-OS-000312-GPOS-00124, SRG-OS-000324-GPOS-00125.

I was adding trying to match the RHEL8 STIG.

Reply:
Let's move forward with this, and we can change it later.